### ISSUE TITLE
info about actually on and off time

### ISSUE THREAD

**Issue by goczen** [biddster/node-red-contrib-schedex#26]
Hi 
Can I read the actually on and off time from this node?
If not, could you develop it?
example
I send the "info" in msg.payload and I get the on time and off time.
Br.
Norbert

**Comment by biddster** [comment]
Seems like a good idea, I'll see what I can do.

**Comment by biddster** [comment]
Any thoughts on how you'd like the dates presented and formatted?

**Comment by biddster** [comment]
Had a quick stab, output message looks like this:

```json
{ 
  "topic": "info", 
  "payload": { 
    "on": "2017-11-29 07:21", 
    "off": "2017-11-29 16:00" }, "
  }
}
```

**Comment by elyobelyob** [comment]
Could we stick to w3 formats, e.g. 1997-07-16T19:20:30+01:00 or similar ..

**Comment by goczen** [comment]
Hi
I tested it, great job.
Could you insert the act. status in the payload obj, too. 
"status": "ON" or "OFF" 
Br.
Norbert

**Comment by biddster** [comment]
I'll look into a state property on the info object. I'll also emit the on/off times in iso 8601 format.

**Comment by elyobelyob** [comment]
Not sure if you saw my quick convo with Knolleary ... he's not thought
about standardising date usage, but suggested Wed, 14 Jun 2017 07:00:00 GMT as
per
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toUTCString

Nick

**Comment by biddster** [comment]
`toUTCString()` is just fine by me.

**Comment by biddster** [comment]
Released on 1.1.0.

**Comment by goczen** [comment]
It is a very useful, Lot of thanks.
